account_invoice_validation_queued/views/queue_job_views.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Copyright 2019 Tecnativa - Pedro M. Baeza
+     License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl). -->
+<odoo>
+    <record id="view_queue_job_account_invoice_validation" model="ir.ui.view">
+        <field name="model">queue.job</field>
+        <field name="priority" eval="9999"/>
+        <field name="arch" type="xml">
+            <tree>
+                <field name="date_created"/>
+                <field name="date_done"/>
+                <field name="state"/>
+                <button type="object"
+                        name="requeue"
+                        string="Requeue"
+                        class="oe_highlight"
+                        attrs="{'invisible': [('state', '!=', 'failed')]}"/>
+                 <button type="object"
+                         name="cancel"
```

Comment:
```suggestion
                         name="unlink"
```

Since the button is invisible in other states, this makes the model method unnecessary.

Reply:
Already answer above